web/beacon-app/src/features/home/components/SetupNewProject.tsx

```diff
@@ -23,19 +23,20 @@ function SetupNewProject() {
         className="min-h-[130px]"
         contentClassName="my-2"
       >
-        <div className="mt-2 flex flex-col justify-between gap-4 px-3 xl:flex-row ">
```

Comment:
`justify-between` was removed so that the button could be closer to the text. However, if you think that the style should be added back, let me know.

Reply:
It may look better to move the button back to the original position because there's so much space its right, but I know that we'll want this component to match the width of the component added above it, so I didn't want to make too many changes here.